kaos/algorithm/coord_conversion.py

```diff
@@ -103,3 +104,39 @@ def lla_to_eci(lat, lon, alt, time_posix):
     eci_vel = Vector3D(loc_eci[1].x.value, loc_eci[1].y.value, loc_eci[1].z.value)
 
     return (eci_pos, eci_vel)
+
+
+def ecef_to_eci(eci_coords, eci_vel, posix_time):
+    """Converts a Cartesian vector in the ECCF to a GCRS frame at the given time.
+
+    Args:
+        eci_coords (tuple): A tuple of the cartesian coordinates of the object in the ECCF frame (m)
+        eci_vel (tuple): A tuple of the velocity of the object in the EECF frame (m/s)
```

Comment:
Roy's suggestion needs to be applied here too

Also, EECF -> ECEF. I'm very impressed at how many different ways you manage to misspell ECEF 😛

Reply:
SAVE ME